epictrack-web/src/components/workPlan/WorkPlanContext.tsx

```diff
@@ -156,13 +159,9 @@ export const WorkplanProvider = ({
 
   const getWorkStatuses = async () => {
     if (workId) {
-      const statusResult = await workService.getWorkStatuses(Number(workId));
+      const statusResult = await statusService.getAll(Number(workId));
       if (statusResult.status === 200) {
-        setStatuses(
-          (statusResult.data as Status[]).sort((a, b) => {
-            return dateUtils.diff(b.posted_date, a.posted_date, "days");
-          })
-        );
+        setStatuses(statusResult.data as Status[]);

```

Comment:
We can specify the type in the service method itself to avoid having to cast here 
![image](https://github.com/bcgov/EPIC.track/assets/91914654/6d1961b5-1782-4810-80e9-a51bd7140c6b)